qt/python/instrumentview/instrumentview/FullInstrumentViewWindow.py

```
@@ -161,71 +185,64 @@ class FullInstrumentViewWindow(QMainWindow):
     _RENDER_MODE_OPTIONS = (_RENDER_MODE_POINTS, _RENDER_MODE_SHAPES_FAST, _RENDER_MODE_RAW_SHAPES)
     _COLOURS = ["#ff7f0e", "#2ca02c", "#d62728", "#9467bd", "#8c564b", "#e377c2", "#7f7f7f", "#bcbd22", "#17becf"]
 
-    def __init__(self, parent=None, off_screen=False):
-        """The instrument in the given workspace will be displayed. The off_screen option is for testing or rendering an image
-        e.g. in a script."""
+    _overlay_meshes = []
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)